.gitignore

```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 #Vagrant
 .vagrant
+
+#idea
+.idea
```

Comment:
What's that?

Reply:
i dont know at which stage this file came across.
![image](https://user-images.githubusercontent.com/83594870/207254297-a86294bf-88e1-4d0e-9ca1-b9e3e68ace3b.png)
![image](https://user-images.githubusercontent.com/83594870/207254325-0a6ec4c8-c8b1-47db-ad37-c07f471688de.png)

Reply:
Does it mean that this change should be on this PR or not?

Reply:
yea why not, added this changed to all my PR if im not mistaken.

Reply:
As for why not, it's because we tend to keep the scope of PRs relevant to what each PR claims to be about.

Few problems with the approach of having it as out-of-scope addition on arbitrary PRs:

- It makes it confusing for code review.
- Suppose that we revert such PR/commit, we will also revert that out of scope change.
- You'll have to rebase all those other PRs once the first one gets merged.

What's the reason for adding it in the first place? Did you add it yourself or was it the IDE that added it? Not including it is not going to have any consequences, unless you accidentally add `.idea` in one of your commits.

Reply:
All in all, if you want it in, just create a small PR that just includes it. Explain there why you want it added, and it can be merged quickly.

Reply:
> All in all, if you want it in, just create a small PR that just includes it. Explain there why you want it added, and it can be merged quickly.

i honestly have no idea how the folder came up, ill just add it in a new PR.

Reply:
Are you using intellij? I'd guess it adds that automatically so you don't push its local configs to your repos.

Reply:
maybe ive opened with pycharm once

Reply:
If you don't need it, then you can also discard that change locally and move on without it.